Dynatrace/dynatrace-operator
# Add fallback for enrichment rules API call

## Description

In case no monitored entity exists, we use the environment scope to get rules configured for the whole environment.


## How can this be tested?

Deploy a dynakube, without activegate, but metadataEnrichment true.
Have enrichment rules set up on your tenant (globally).
EnrichmentRules should be in the DynaKube's status
